### PR TITLE
update grate-overlay

### PR DESCRIPTION
**Pull request by hedmo**
i updated README.md to

### CODE REVIEW

File: x11-drivers/xf86-video-opentegra/xf86-video-opentegra-9999.ebuild [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-inherit xorg-2
+inherit xorg-3
 
 EGIT_REPO_URI="https://github.com/grate-driver/xf86-video-opentegra.git"
 
 DESCRIPTION="X.Org video driver for NVIDIA Tegra20+"
 HOMEPAGE="https://github.com/grate-driver"
 
-KEYWORDS="arm"
+KEYWORDS=""
```
Comment (digetx) [inline x11-drivers/xf86-video-opentegra/xf86-video-opentegra-9999.ebuild:13]: Does this mean that it's now necessary to manually unmask the keywords?

Comment (hedmo) [reply x11-drivers/xf86-video-opentegra/xf86-video-opentegra-9999.ebuild:13]: Yes. Gentoo DO not keyword live ebuilds. If you want that i need to make a version ebuild...

Comment (hedmo) [reply x11-drivers/xf86-video-opentegra/xf86-video-opentegra-9999.ebuild:13]: And that is how i want it . Why Always download THE same git ..... If we where updating often that is another story..

Comment (digetx) [reply x11-drivers/xf86-video-opentegra/xf86-video-opentegra-9999.ebuild:13]: Alright, thank you for the clarification.

File: sys-config/grate/grate-1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="A configuration for portage to make the insatll from grate easy"
```
Comment (digetx) [inline sys-config/grate/grate-1.0.ebuild]: s/insatll/install

Comment (hedmo) [reply sys-config/grate/grate-1.0.ebuild]: DO you want . version ebuilds to?

Comment (digetx) [reply sys-config/grate/grate-1.0.ebuild]: Not in this PR, maybe later. I'll probably need to issue release versions first.

Comment (hedmo) [reply sys-config/grate/grate-1.0.ebuild]: okej . then i can merge it ?

Comment (hedmo) [reply sys-config/grate/grate-1.0.ebuild]: > Not in this PR, maybe later. I'll probably need to issue release versions first.

the release versions will be the latest git release you made in a zip and have the version name from Xorg.0.log + plus a date like my grate-sources: https://github.com/hedmo/hedmos-overlay/blob/master/sys-kernel/grate-sources/grate-sources-5.12.0_p20210504.ebuild

Comment (digetx) [reply sys-config/grate/grate-1.0.ebuild]: The PR looks good to me, please merge it. I'll test it all during the next days. Thank you very much!

The git-release is a good option, I have no objections to it. Please feel free to add it!

**Pull request merged**